app/helpers/api_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'pandoc-ruby'
+module ApiHelper
+  def export_organizations_as_docx organizations
+    markdown = export_organizations_as_markdown( organizations )
+    converter = PandocRuby.new(markdown, :from => :markdown, :to => :docx)
+    result = converter.convert
+    return result
+  end
+
+  def export_organizations_as_markdown organizations
+    template = File.read(File.join(Rails.root,'templates/organizations_template.markdown.erb'))
```

Comment:
Line is too long. [95/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.